CMakeLists.txt

```diff
@@ -431,10 +433,14 @@ endif()
 
 # ---[ OpenMP
 if(USE_OPENMP)
+  if (NOT OPENMP_FLAVOR)
```

Comment:
Where is OPENMP_FLAVOR from?

Reply:
From cmake command line. Is the selector as requested.